cmd/internal/flowmode/cmd_run.go

```diff
@@ -346,15 +351,50 @@ func getEnabledComponentsFunc(f *flow.Flow) func() map[string]interface{} {
 	}
 }
 
-func loadFlowFile(filename string) (*flow.File, error) {
-	bb, err := os.ReadFile(filename)
+func loadFlowSource(path string) (*flow.Source, error) {
```

Comment:
The contents of this feels like it may belong in the pkg/flow space with a light wrapper here like it prior. This would also make us 1 step closer to other things (ie. modules) reusing this logic.